apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { User } from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getCitizenshipStatusesAdmin,
+  getLanguage,
+  getProvinceOrTerritory,
+} from "@gc-digital-talent/i18n";
+import { Link, Well } from "@gc-digital-talent/ui";
+
+interface ProfileDetailsProps {
+  user: User;
+}
+
+const ProfileDetails = ({ user }: ProfileDetailsProps) => {
+  const intl = useIntl();
+
+  const location = (
+    <p>
+      {user.currentCity || intl.formatMessage(commonMessages.notFound)},{" "}
+      {intl.formatMessage(
+        user.currentProvince
+          ? getProvinceOrTerritory(user.currentProvince)
+          : commonMessages.notFound,
+      )}
+    </p>
+  );
+
+  const telephone = (
+    <p>
+      {user.telephone ? (
+        <Link
+          external
+          href={`tel:${user.telephone}`}
+          aria-label={user.telephone.replace(/.{1}/g, "$& ")}
+        >
+          {user.telephone}
+        </Link>
+      ) : (
+        ""
+      )}
+    </p>
+  );
```

Comment:
This is resuling in an empty paragraph tag. Could we check the phone number outside and return `null` if it doesn't exist instead?

Reply:
Fixed by [d9a30c7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8853/commits/d9a30c77834f2227e1b7162be5f836d01b529e67)